templates/careers/index.html

```diff
@@ -18,82 +18,109 @@
   <img src='https://pubads.g.doubleclick.net/activity;xsp=5024608;ord=1?' width=1 height=1 border=0>
 </noscript>
 
-<section class="p-strip--image is-dark is-deep" style="background-image: url(https://assets.ubuntu.com/v1/aebc1742-01_suru2_dark_2K.png); background-position: 100% 55%; background-size: 180%; background-repeat: repeat-x;">
-  <div class="row">
-    <div class="col-8 col-start-large-4">
-      <h1 class="p-heading--2">Make a difference in open source</h1>
-      {% if vacancies %}
-        {% include "careers/_search.html" %}
-      {% endif %}
+<div class="is-paper">
+  <section>
+    <div class="u-fixed-width">
+      <h1 class="p-text--x-small-capitalised">Careers</h1>
+      <hr class="p-rule">
     </div>
-  </div>
-</section>
+    <div class="row">
+      <div class="col-9 col-medium-4 col-start-large-4">
+        <div class="p-block">
+        <h2 class="u-no-margin--bottom p-heading--1">Passionate about open source?</h2>
+          <h3 class="p-heading--2">So are we. Be where the cutting edge is<br class="u-hide--small" /> established.</h3>
+        </div>
+        {% if vacancies %}
+          {% include "careers/_search.html" %}
+        {% endif %}
+      </div>
+      <div class="col-12">
+        <div class="u-fixed-width p-strip--departments"></div>
```

Comment:
If it is inside `col-12` it doesn't need `u-fixed-width`, but also why is it a strip without content?

Shouldn't the whole section be inside strip with suru background?

Reply:
@lyubomir-popov asked to add this background as a separate element.

Reply:
OK. Before we have those properly in Vanilla I guess it's fine as long as it works.

Naming seems a bit weird though. `p-strip--depatrments` makes it look like it's a strip style, and it's just a decorative suru background (that is not meant to have any content?).

Reply:
Lyubo ammended and improved this with a separate suru